learn/core_concepts/indexes.md

```diff
@@ -130,3 +131,11 @@ Typo tolerance is a built-in feature that helps you find relevant results even w
 You can update the typo tolerance settings using the [update settings endpoint](/reference/api/settings.md#update-settings) or the [update typo tolerance endpoint](/reference/api/typo_tolerance.md#update-typo-tolerance).
 
 [Learn more about typo tolerance](/learn/configuration/typo_tolerance.md)
+
+### Faceting
+
+Facets are a specific use-case of filters in Meilisearch, whether something is a facet or filter depends on your UI and UX design. Like filters, you need to add your facets to [`filterableAttributes`](/reference/api/filterable_attributes.md#update-filterable-attributes) then, make a search query using the [`filter` search parameter](/reference/api/search.md#filter).
```

Comment:
A few punctuation changes.

The first comma is not strong enough. I think a colon works better because the longer pause establishes a "statement: explanation" structure.

The second comma I changed ("[…] add your facets to filterableAttributes then, […]") was, AFAIK, wrong. The pause is very weird if you say it out loud: "do this then, that".

```suggestion
Facets are a specific use-case of filters in Meilisearch: whether something is a facet or filter depends on your UI and UX design. Like filters, you need to add your facets to [`filterableAttributes`](/reference/api/filterable_attributes.md#update-filterable-attributes), then make a search query using the [`filter` search parameter](/reference/api/search.md#filter).
```